com.civicrm.hrjobroles/js/src/job-roles/controllers/hr-job-roles-controller.js

```diff
@@ -1,11 +1,13 @@
 define([
+    'common/angular',
     'job-roles/controllers/controllers',
+    'common/moment',
     'common/filters/angular-date/format-date'
-], function (controllers) {
+], function (angular, controllers, moment) {
     'use strict';
 
-    controllers.controller('HRJobRolesController', ['$scope', 'HR_settings', '$log', '$routeParams', 'HRJobRolesService', '$route', '$timeout', '$filter', 'DateValidationService', 'DateFactory',
-        function ($scope, HR_settings, $log, $routeParams, HRJobRolesService, $route, $timeout, $filter, DateValidationService, DateFactory) {
+    controllers.controller('HRJobRolesController', ['$scope', 'HR_settings', '$log', '$routeParams', 'HRJobRolesService', '$route', '$timeout', '$filter', 'DateValidation', '$q',
+        function ($scope, HR_settings, $log, $routeParams, HRJobRolesService, $route, $timeout, $filter, DateValidation, $q) {
```

Comment:
Just to have some kind of consistent approach, let's have the angular built-in components listed first, and then our custom ones:
```js
controllers.controller('HRJobRolesController', ['$filter', '$log', '$q', '$route', '$routeParams', '$scope', '$timeout', 'DateValidation', 'HR_settings', 'HRJobRolesService',
    function ($filter, $log, $q, $route, $routeParams, $scope, $timeout, DateValidation, HR_settings, HRJobRolesService) {
        // ...
    }
]);
```